app/views/shared/_citations.html.erb

```diff
@@ -19,7 +19,7 @@
   <% @presenter.creator.each do |creator| %>
   <meta name="citation_author" content="<%= creator %>"/>
   <% end %>
-  <meta name="citation_publication_date" content="<%= @presenter.date_created.first %>"/>
+  <meta name="citation_publication_date" content="<%= @presenter.date_created.first unless @presenter.date_created.nil? %>"/>
```

Comment:
Hi @jcoyne - :date_created is delegated to :solr_document in the FileSetPresenter. What are you suggesting would be in the returned array when  :sol_document has no date_created? Thanks.